src/macros.rs

```diff
@@ -1384,15 +1439,20 @@ fn format_lazy_static(
             "{}static ref {}: {} =",
             vis,
             id,
-            ty.rewrite(context, nested_shape)?
+            ty.rewrite_result(context, nested_shape)?
         ));
-        result.push_str(&rewrite_assign_rhs(
-            context,
-            stmt,
-            &*expr,
-            &RhsAssignKind::Expr(&expr.kind, expr.span),
-            nested_shape.sub_width(1)?,
-        )?);
+        result.push_str(
+            &rewrite_assign_rhs(
```

Comment:
Question: Do we have plans to make `rewrite_assign_rhs` return a `RewriteResut`?

Reply:
Definitely yes. However, when I tried it a few weeks ago, I found that we might not be able to fully avoid using `unknown_error()` inside `rewrite_assign_rhs` due to challenges in grabbing the correct span. Anyway, I’ll give it another try and update you on the outcome.